corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -82,6 +82,22 @@
 
   <div id="report-datagrid"
        class="ko-template">
+
+    <div class="datagrid-save">
+      <button type="button"
+              data-bind="click: exportData"
+              class="btn btn-default">
+        <i class="fa fa-download"></i>
+        {% trans "Export Data" %}
+      </button>
+      <button type="button"
+              data-bind="click: saveFilters"
+              class="btn btn-default">
+        <i class="fa fa-save"></i>
```

Comment:
Nitpick: we have a lot of save buttons but very few have an icon. Do people still know what floppy disks are?